dll/cpl/sysdm/general.c

```diff
@@ -598,73 +604,85 @@ static VOID GetSystemVersion(HWND hwnd)
     HeapFree(GetProcessHeap(), 0, pwszStr);
 }
 
-ULONGLONG GetSecondsQPC(VOID)
+/**
+ * @brief
+ * An equivalent of GetTickCount64, implemented using QueryPerformanceCounter.
+ *
+ * @return
+ * The number of milliseconds that have elapsed since the system was started.
+ **/
```

Comment:
```suggestion
 */
```

Reply:
Not really; it indicates the closing of the opened `/**` block.

Reply:
I checked other places where these kind of annotations are used, in most places it has only opening `/**` and closing `*/`

Reply:
Also this is how it's in our documentation: https://reactos.org/wiki/Documentation_Guidelines#API_Documentation